tests/snuba/search/test_backend.py

```diff
@@ -115,6 +115,31 @@ def setUp(self):
             is_active=False,
         )
 
+        self.project2 = self.create_project(organization=self.project.organization)
```

Comment:
Could create these as `@fixture`s since they're not used in every test

Reply:
maybe i'm misunderstanding, but wouldn't i then need to do something to access the fixtures before querying to force creation?

Reply:
That's a good point... you could declare the groups like `expected = [self.group1, self.group_p2, self.group2]` before you make the call. Not sure how worthwhile it is, but in general I like trying to limit how much database work we're doing in tests.

Reply:
i just moved into a separate `set_up_multi_project` function to be more explicit